.github/workflows/rust.yml

```diff
@@ -0,0 +1,19 @@
+name: Rust
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Clippy check
+        working-directory: ./rusthelloserverless
+        run: cargo clippy --verbose
```

Comment:
I think we won't be able to test this workflow until merged.

Are you sure cargo is avail without using `actions-rust-lang/setup-rust-toolchain@v1`? https://github.com/actions-rust-lang/setup-rust-toolchain

Reply:
Yes, it will be available. For a simple `cargo check` there is no need to setup a specific toolchain. The current workflow is based on the default Rust workflow template that GitHub offers, so it should work. We have a very similar workflow setup for our `cpp-rust-driver`: https://github.com/scylladb/cpp-rust-driver/blob/master/.github/workflows/cargo_check.yml